src/compiler/parser.ts

```diff
@@ -2122,11 +2122,11 @@ namespace Parser {
     }
 
     function scanError(message: DiagnosticMessage, length: number): void {
-        parseErrorAtPosition(scanner.getTextPos(), length, message);
+        parseErrorAtPosition(scanner.getTokenEnd(), length, message);
     }
 
     function getNodePos(): number {
-        return scanner.getStartPos();
+        return scanner.getTokenFullStart();

```

Comment:
this is unexpected; I would have thought a node would only include its own span, not the whitespace/trivia in front of it.
changing it to getTokenStart breaks every test ever, though -- symbol baselines of course, emit now misses leading comments, and some types are missed somehow.